docs/events/indicators-of-compromise.asciidoc

```diff
@@ -0,0 +1,95 @@
+** *{filebeat} integrations* - If you are collecting indicator data from {filebeat} threat intelligence integrations, indicator documents are stored in the `filebeat-*` index pattern.
+* Make sure the indicator data you're ingesting is mapped to {ecs-ref}[Elastic Common Schema (ECS)].
+
+[discrete]
+[[intelligence-page-ui]]
+== Intelligence page UI
+
+After you add indicators to the Indicators page, you can <<examine-indicator-details, examine>>, search, filter, and take action on indicator data. Indicators also appear in the Trend view, which shows the occurrence of indicators over time.
+
+[role="screenshot"]
+image::images/interact-with-indicators-table.gif[width=90%][height=90%][Shows how to interact with the Intelligence page]
+
+[discrete]
+[[examine-indicator-details]]
+=== Examine indicator details
+Learn more about an IoC by clicking *View details*, then opening the Indicator details flyout. The flyout contains these informational tabs:
+
+* *Overview*: Provides a summary of the indicator, including the indicator's name, the threat intelligence feed it came from, the indicator type, and additional relevant data.
++
+NOTE: Some threat intelligence feeds provide  https://www.cisa.gov/tlp#:~:text=Introduction,shared%20with%20the%20appropriate%20audience[Traffic Light Protocol (TLP) markings]. The `TLP Marking` field will be empty if the data is not available. The `Confidence` field will also be empty if the data is not available.
+
+* *Table*: The indicator data in table format.
+* *JSON*: The indicator data in JSON format.
++
+[role="screenshot"]
+image::images/indicator-details-flyout.png[Shows the Indicator details flyout, 600]
+
+[discrete]
+[[find-related-sec-events]]
+== Find related security events
+
+Investigating an indicator in <<timelines-ui,Timeline>> helps you find related security events in your environment. You can add an indicator to Timeline from the Indicators table or the Indicator details flyout.
+
+[role="screenshot"]
+image::images/indicator-in-timeline.png[Shows the results of an indicator being investigated in Timeline]
+
+When you add an indicator to Timeline, a new Timeline opens with a pre-populated KQL query. The query contains the indicator field and value pair that you selected plus the field and value pair of the mapped source event.
+
+For example, imagine you've added this file hash indicator to Timeline:
+
+`threat.indicator.file.hash.sha256 : c207213257a63589b1e1bd2f459b47becd000c1af8ea7983dd9541aff145c3ba`
+
+A new Timeline opens with an automatically populated KQL query. The query contains the indicator field value pair (mentioned previously) and the mapped source event field value pair, which is:
+
+`file.hash.sha256 : c207213257a63589b1e1bd2f459b47becd000c1af8ea7983dd9541aff145c3ba`.
+
+Using a KQL query that includes both the indicator and source event allows Timeline to find all events and alerts that have matching field and value pairs.
```

Comment:
The "pairs" here get called both "field and value pairs" and "field value pairs," but I think "[field-value pairs](https://en.wikipedia.org/wiki/Name%E2%80%93value_pair)" is more standard. 
```suggestion
When you add an indicator to Timeline, a new Timeline opens with a pre-populated KQL query. The query contains the indicator field-value pair that you selected plus the field-value pair of the mapped source event.

For example, imagine you've added this file hash indicator to Timeline:

`threat.indicator.file.hash.sha256 : c207213257a63589b1e1bd2f459b47becd000c1af8ea7983dd9541aff145c3ba`

A new Timeline opens with an automatically populated KQL query. The query contains the indicator field-value pair (mentioned previously) and the mapped source event field-value pair, which is:

`file.hash.sha256 : c207213257a63589b1e1bd2f459b47becd000c1af8ea7983dd9541aff145c3ba`.

Using a KQL query that includes both the indicator and source event allows Timeline to find all events and alerts that have matching field-value pairs.
```